src/lightning/pytorch/loggers/comet.py

```diff
@@ -166,24 +164,22 @@ def __init__(self, *args, **kwarg):
         - `Comet Documentation <https://www.comet.com/docs/v2/integrations/ml-frameworks/pytorch-lightning/>`__
 
     Args:
-        api_key: Required in online mode. API key, found on Comet.ml. If not given, this
+        api_key: Required in online mode. API key, found on Comet.com. If not given, this
             will be loaded from the environment variable COMET_API_KEY or ~/.comet.config
             if either exists.
         save_dir: Required in offline mode. The path for the directory to save local
             comet logs. If given, this also sets the directory for saving checkpoints.
         project_name: Optional. Send your experiment to a specific project.
-            Otherwise will be sent to Uncategorized Experiments.
-            If the project name does not already exist, Comet.ml will create a new project.
-        rest_api_key: Optional. Rest API key found in Comet.ml settings.
-            This is used to determine version number
-        experiment_name: Optional. String representing the name for this particular experiment on Comet.ml.
+            Otherwise, will be sent to Uncategorized Experiments.
+            If the project name does not already exist, Comet.com will create a new project.
+        experiment_name: Optional. String representing the name for this particular experiment on Comet.com.
         experiment_key: Optional. If set, restores from existing experiment.
         offline: If api_key and save_dir are both given, this determines whether
             the experiment will be in online or offline mode. This is useful if you use
             save_dir to control the checkpoints directory and have a ~/.comet.config
             file but still want to run offline experiments.
         prefix: A string to put at the beginning of metric keys.
-        \**kwargs: Additional arguments like `workspace`, `log_code`, etc. used by
+        **kwargs: Additional arguments like `workspace`, `log_code`, etc. used by
             :class:`CometExperiment` can be passed as keyword arguments in this logger.
```

Comment:
You can reuse the same docstring as `comet_ml.start` for most of those. You can also takes inspiration from the torchtune docstring https://github.com/pytorch/torchtune/blob/main/torchtune/utils/metric_logging.py#L332-L349